src/service/paymentService.js

```diff
@@ -1,40 +1,42 @@
 import _ from 'lodash';
```

Comment:
Noticed that you don't have handle error with try catch. Please do so and throw error massage and error code accordingly

Reply:
will be update on yujith's fork.